styles.css

```diff
@@ -1,90 +1,123 @@
-body {
-  display: flex;
+/* Typography and Text Alignment */
+h1,
+h2,
+p {
+  font-family: cursive;
+  text-align: center;
+}
+
+/* Wrapper */
+.wrapper {
+  min-height: 300px;
+}
+
+/* Main Section (Add Book Form) */
+.main {
+  display: none;
   flex-direction: column;
+  justify-content: space-between;
   align-items: center;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
   justify-content: center;
-  min-height: 100vh;
-  margin: 0;
-  background-color: #f4f4f4;
+  align-items: center;
 }
 
-h1.main-header {
-  font-size: 28px;
-  margin-bottom: 20px;
+/* Book List (Shelf) */
+.book {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px; /* Increase padding for row size */
+  margin: 10px auto; /* Center horizontally and increase margin for spacing */
+  border: 1px solid #2d2d2d;
+  background-color: rgb(230, 230, 230);
+  width: 85%; /* Use 85% width for responsiveness */
 }
 
-h2.book-list-header {
-  font-size: 24px;
-  margin-bottom: 10px;
+.remove-button {
+  font-family: 'Poppins', cursive;
+  border: 1px solid #2d2d3d;
+  box-shadow: 3px 5px;
+  background-color: white;
+  cursor: pointer;
 }
 
-#bookListContainer {
-  width: 80%;
+/* Buttons */
+#addButton {
+  font-family: 'Poppins', cursive;
   background-color: white;
-  padding: 20px;
-  border-radius: 8px;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-  margin-bottom: 20px;
+  border: 1px solid #2d2d2d;
+  box-shadow: 3px 5px;
 }
 
-form {
+/* Navigation Bar */
+.nav-bar {
+  margin: 5%;
+  margin-bottom: 1%;
+  padding-left: 5%;
+  padding-right: 5%;
+  border: 1px solid #2d2d2d;
   display: flex;
-  flex-direction: column;
-  align-items: flex-start;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
 }
 
-label {
-  margin-top: 10px;
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
 }
 
-input[type="text"] {
-  width: 100%;
-  padding: 5px;
-  margin-top: 5px;
+.nav-item {
+  font-family: 'Poppins', cursive;
+  padding: 5%;
+  min-width: max-content;
+  cursor: pointer;
+  text-align: center;
 }
 
-button {
-  margin-left: 78%;
-  margin-top: 10px;
-  padding: 5px 15px;
-  background-color: #007bff;
-  color: white;
-  border: none;
-  cursor: pointer;
+li {
+  text-align: center;
+  padding-top: 5%;
 }
 
-ul {
-  list-style: none;
-  padding: 0;
-  margin: 0;
+#logo {
+  cursor: pointer;
 }
 
-li {
-  border-bottom: 1px solid #ccc;
-  padding: 10px;
-  width: 100%;
+/* Footer */
+footer {
   display: flex;
-  justify-content: space-between;
-  align-items: center;
+  flex-direction: row;
+  width: 80%;
+  margin: 5%;
+  padding-left: 5%;
+  padding-right: 5%;
+  border: 1px solid #2d2d2d;
 }
 
-li:nth-child(even) {
-  background-color: #f4f4f4;
+/* Sections Display */
+#listSection {
+  display: none;
+}
 
-  li:nth-child(odd) {
-    background-color: white;
-  }
+.contact {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- [ ] The contact section should look like the wireframe it is a project requirement for this milestone, please edit the styles to be similar to the wireframe 👇🏽 

| Wireframe | Your design |
|---|---|
| ![image](https://github.com/Sami-Ullah-Tufail/Awesome-books/assets/58642949/98e586a1-bc93-42f8-8df6-3be7138d0cdf) | ![image](https://github.com/Sami-Ullah-Tufail/Awesome-books/assets/58642949/7448db72-44c9-4f47-b43d-96ae6d74e60e) |